src/app/admin/create-entity/create-entity.component.ts

```diff
@@ -1596,7 +1597,7 @@ export class CreateEntityComponent implements OnInit, AfterContentChecked {
 
             if (i == this.usecaseSchema.length - 1 && !this.errArr.length) {
               this.getEntityProperties();
-              this.nextStep();
+              moveNext?this.nextStep():'';
```

Comment:
@Prasang023 Please add space, its not recognize easily, it's seems like one sentence,